### DISCUSSION TITLE
Zigbee2mqtt connect to MQTT but no mosquitto log / autodiscovery

### DISCUSSION THREAD

**Discussion by matteodallefeste** [Koenkk/zigbee2mqtt#15034] [General]
Hi,
I installed Zigbee2MQTT on Home assistant.
Zigbee2MQTT start and i paired 3 Aqara door sensors.

Inside of Zigbee2MQTT Logs i see a successful connection to MQTT and i see the Door sensor react.
I configured Home Assistant MQTT to log in debug mode and i see other MQTT devices working but i don't see the Zigbee2MQTT messages.

LOG FROM ZIGBEE2MQTT:

 `[20:04:50] INFO: Preparing to start...
[20:04:54] INFO: Socat not enabled
[20:05:05] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2022-11-16 20:05:30: Logging to console and directory: '/config/zigbee2mqtt/log/2022-11-16.20-05-29' filename: log.txt
Zigbee2MQTT:info  2022-11-16 20:05:30: Starting Zigbee2MQTT version 1.28.2 (commit #unknown)
Zigbee2MQTT:info  2022-11-16 20:05:30: Starting zigbee-herdsman (0.14.68)
Zigbee2MQTT:info  2022-11-16 20:05:34: zigbee-herdsman started (resumed)
Zigbee2MQTT:info  2022-11-16 20:05:34: Coordinator firmware version: '{"meta":{"maintrel":"9 ","majorrel":"6","minorrel":"7","product":8,"revision":"6.7.9.0 build 405"},"type":"EZSP v8"}'
Zigbee2MQTT:info  2022-11-16 20:05:34: Currently 3 devices are joined:
Zigbee2MQTT:info  2022-11-16 20:05:34: 0x00158d000847f91f (0x00158d000847f91f): MCCGQ11LM - Xiaomi Aqara door & window contact sensor (EndDevice)
Zigbee2MQTT:info  2022-11-16 20:05:34: 0x00158d00084812eb (0x00158d00084812eb): MCCGQ11LM - Xiaomi Aqara door & window contact sensor (EndDevice)
Zigbee2MQTT:info  2022-11-16 20:05:34: 0x00158d0008065203 (0x00158d0008065203): MCCGQ11LM - Xiaomi Aqara door & window contact sensor (EndDevice)
Zigbee2MQTT:info  2022-11-16 20:05:34: Zigbee: disabling joining new devices.
Zigbee2MQTT:info  2022-11-16 20:05:35: Connecting to MQTT server at mqtt://core-mosquitto:1883
Zigbee2MQTT:info  2022-11-16 20:05:35: Connected to MQTT server
Zigbee2MQTT:info  2022-11-16 20:05:35: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload '{"state":"online"}'
Zigbee2MQTT:info  2022-11-16 20:05:36: Started frontend on port 0.0.0.0:8099
Zigbee2MQTT:info  2022-11-16 20:05:36: MQTT publish: topic 'homeassistant/sensor/0x00158d000847f91f/battery/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x00158d000847f91f"],"manufacturer":"Xiaomi","model":"Aqara door & window contact sensor (MCCGQ11LM)","name":"porta_studio"},"device_class":"battery","enabled_by_default":true,"entity_category":"diagnostic","name":"porta_studio_battery","state_class":"measurement","state_topic":"zigbee2mqtt/0x00158d000847f91f","unique_id":"0x00158d000847f91f_battery_zigbee2mqtt","unit_of_measurement":"%","value_template":"{{ value_json.battery }}"}'
`
I reinstalled the MQTT integration as described in some post found but i did not resolve.
My configurations:

`mqtt:
  server: mqtt://core-mosquitto:1883
  user: xxxx
  password: yyyyy
  keepalive: 60
  reject_unauthorized: false
  version: 4
serial:
  port: /dev/ttyUSB1
  adapter: ezsp
frontend:
  port: 8099
advanced:
  homeassistant_legacy_entity_attributes: false
  legacy_api: false
  legacy_availability_payload: false
device_options:
  legacy: false
devices:
  '0x00158d000847f91f':
    friendly_name: '0x00158d000847f91f'
    description: Ingresso Studio
    homeassistant:
      name: porta_studio
    legacy: false
    optimistic: true
  '0x00158d00084812eb':
    friendly_name: '0x00158d00084812eb'
    description: Ingresso appartamento
    homeassistant:
      name: porta_appartamento
    legacy: false
    optimistic: true
  '0x00158d0008065203':
    friendly_name: '0x00158d0008065203'
    description: Porta esterna
    homeassistant:
      name: porta_esterna
    legacy: false
    optimistic: true
    filtered_attributes: []
`

NOTE: in the configuration i see that Zigbee2MQTT continue to change me the username to "addon" and set a long self generated password. 

Anybody can help me please?
Thanks
Matteo

**Comment by jdhgmann04** [comment]
This started happening to me on all aqara devices after updating to 1.28 and ha to 11.2. Here is what i found seems to fix it.

https://github.com/Koenkk/zigbee2mqtt/discussions/15060#discussioncomment-4179376

**Comment by matteodallefeste** [comment]
Just resolved reinstalling Zigbee2Mqtt Edge instead of the current release1.28.

 I deleted the entire configuration file and re entered mqtt, devices  and serial minimal params.

If i retry to reinstall  the 1.28 the problem return.

I hope this could help friends in my same situation ;)
